migration/migrating-4-4/migrating-openshift-4_2-to-4.adoc

```diff
@@ -13,20 +13,19 @@ The CAM tool's web console and API, based on Kubernetes custom resources, enable
 .Prerequisites
 
 * You must have xref:../../cli_reference/openshift_cli/administrator-cli-commands.html#policy[`cluster-admin` privileges] on all clusters.
-* You must have a replication repository that supports the S3 API and is accessible to the source and target clusters.
+* You must configure a replication repository that supports the S3 API and is accessible to the source and target clusters.
 
 include::modules/migration-understanding-cam.adoc[leveloffset=+1]
 
 [id='Configuring-replication-repository_{context}']
 == Configuring a replication repository
 
-You must configure a replication repository. The CAM tool uses the replication repository as a temporary intermediate storage space for copying data from the source cluster to the target cluster, using the xref:migration-understanding-data-copy-methods_{context}[file system or snapshot copy method].
+You must configure S3 storage as a replication repository. The CAM tool copies data from the source cluster to the replication repository, and from there to the target cluster, using either the xref:migration-understanding-data-copy-methods_{context}[file system or the snapshot data copy method].
```

Comment:
Change to:

You must configure an Object Store to use as a replication repository. The CAM tool copies data from the source cluster to the replication repository, and from there to the target cluster, using either the xref:migration-understanding-data-copy-methods_{context}[file system or the snapshot data copy method].